.circleci/config.yml

```diff
@@ -85,6 +82,18 @@ jobs:
               exit 1
             fi
 
+  run-staging-report:
+    circleci_ip_ranges: true
+    docker:
+      - image: cimg/go:1.23.1
+    steps:
+      - checkout
+      - run:
+          name: Run staging report
+          command: |
+            cd ops
+            go run ./cmd/print_staging_report/main.go
```

Comment:
This command has required args for `GithubTokenFlag`, `GitSHAFlag` and `GithubRepoFlag` but I don't see them passed here. Won't that cause this line to fail?

Reply:
They are injected via CCI env vars since they are secrets.